src/transformState.js

```diff
@@ -4,8 +4,40 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const action in actions) {
```

Comment:
Actions is an array, you can iterate through, using for...of loop, in that way you wouldn't need to use indexes to find the right action in your loop